code/development-env/README.md

```diff
@@ -2,43 +2,20 @@
 
 ## Initial set-up
 
-### Using docker-machine (Windows)
+### Start MongoDB and Vault docker containers
```

Comment:
The contents of this heavily overlap with the contents of the README that is here which Alex and Andy have just made a lot of updates to - https://github.com/NERC-CEH/datalab/blob/master/code/development-env/docker/README.md

Could you please review them to build a single unified document. Best to move additional content from this version into the other document. We may then want to move the README back to this location but it will probably be easier to review if we do it that way.